src/m365/spo/commands/listitem/listitem-batch-add.ts

```diff
@@ -162,7 +171,12 @@ class SpoListItemBatchAddCommand extends SpoCommand {
       },
       data: requestBody.join('')
     };
-    await request.post(requestOptions);
+    const response: any = await request.post(requestOptions);
+    const parsedResponse = this.parseBatchResponseBody(response);
+
+    if (parsedResponse.some(r => r.HasException)) {
+      throw `Some items were not created successfully due to the following errors: ${os.EOL}${parsedResponse.filter(f => f.HasException).map(f => { return `- Line ${f.csvLineNumber}: ${f.FieldName} - ${f.ErrorMessage}`; }).join(os.EOL)}`;
```

Comment:
This error message is unclear. Let's rephrase to: "Creating some items has failed with the following errors"